src/OpenTelemetry.Contrib.Extensions/OpenTelemetry.Contrib.Extensions.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(Configuration)' == 'Debug'">$(TargetFrameworks);net5.0</TargetFrameworks> <!-- Added just to get proper nullable analysis in IDE -->
+    <Description>OpenTelemetry .NET SDK Extensions</Description>
+    <MinVerTagPrefix>Extensions-</MinVerTagPrefix>
+    <Nullable>enable</Nullable>
+    <EnableAnalysis>true</EnableAnalysis>
+    <EnablePublicApi>true</EnablePublicApi>
```

Comment:
Projects in this repo can now opt into static analysis and/or public api tracking using these 2 props.